sqlserver/datadog_checks/sqlserver/database_metrics/database_agent_metrics.py

```diff
@@ -0,0 +1,159 @@
+from datadog_checks.base.config import is_affirmative
+
+from datadog_checks.sqlserver.const import  AWS_RDS_HOSTNAME_SUFFIX
+
+from .base import SqlserverDatabaseMetricsBase
+
+AGENT_ACTIVITY_DURATION_QUERY = {
```

Comment:
does it make sense to have 2 separate PRs? These 2 features seem to be independent with each other. split into 2 PRs will make it more reviewable and maintainable. 
- agent activity metrics
- agent job history